src/Builder/Creator/MethodStatementsCreator.php

```diff
@@ -0,0 +1,165 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+declare(strict_types=1);
+
+namespace SprykerSdk\Integrator\Builder\Creator;
+
+use PhpParser\BuilderFactory;
+use PhpParser\Node\Expr\ArrayItem;
+use PhpParser\Node\Expr\ConstFetch;
+use PhpParser\Node\Expr\Variable;
+use PhpParser\Node\Identifier;
+use PhpParser\Node\Name;
+use PhpParser\Node\Scalar\String_;
+use SprykerSdk\Integrator\Transfer\ClassInformationTransfer;
+
+class MethodStatementsCreator extends AbstractMethodCreator implements MethodStatementsCreatorInterface
+{
+    /**
+     * @var int
+     */
+    protected const SIMPLE_VARIABLE_SEMICOLON_COUNT = 1;
+
+    /**
+     * @param \SprykerSdk\Integrator\Transfer\ClassInformationTransfer $classInformationTransfer
+     * @param mixed $value
+     *
+     * @return array
+     */
+    public function createMethodStatementsFromValue(ClassInformationTransfer $classInformationTransfer, $value): array
+    {
+        if (is_array($value)) {
+            return $this->createNodeTreeFromArrayValue($classInformationTransfer, $value);
+        }
+
+        return $this->createNodeTreeFromStringValue($classInformationTransfer, $value);
+    }
+
+    /**
+     * @param \SprykerSdk\Integrator\Transfer\ClassInformationTransfer $classInformationTransfer
+     * @param mixed $value
+     *
+     * @return array
+     */
+    protected function createNodeTreeFromStringValue(ClassInformationTransfer $classInformationTransfer, $value): array
+    {
+        $arrayItems = [];
+        $valueItems = explode(
+            '::',
+            $this->getShortClassNameAndAddToClassInformation($classInformationTransfer, $value),
+        );
+        $arrayItems[] = new ArrayItem(
+            $this->createClassConstantExpression($valueItems[0], $valueItems[1]),
+        );
+
+        return $arrayItems;
+    }
+
+    /**
+     * @param \SprykerSdk\Integrator\Transfer\ClassInformationTransfer $classInformationTransfer
+     * @param mixed $value
+     *
+     * @return array
+     */
+    protected function createNodeTreeFromArrayValue(ClassInformationTransfer $classInformationTransfer, $value): array
+    {
+        $arrayItems = [];
+        foreach ($value as $key => $item) {
+            $insideArrayItems = [];
+            $itemParts = [];
+
+            if (is_string($item)) {
+                $item = $this->getShortClassNameAndAddToClassInformation($classInformationTransfer, $item);
+                $itemParts = explode('::', $item);
+            } else {
```

Comment:
it makes no sense to keep this else part. We are insideArrayItems var only if item is array on the line 91